Applications/Classification/res/Classification.ini

```diff
@@ -33,4 +33,4 @@ Type = OutputLayer
 Id = 1
 HiddenSize = 10		# Output Layer Dimension ( = Weight Width )
 Bias_zero = true
-Activation = sigmoid 	# activation : sigmoid, softmax
+Activation = softmax 	# activation : sigmoid, softmax
```

Comment:
Looks good. One quesiton. Are candidate activation functions just two functions? Or Can another function (e.g., Relu, Weaky Relu) can be used?

Reply:
For now, relu, tanh, sigmoid, softmax are available.